tensorflow_estimator/python/estimator/tpu/tpu_config.py

```diff
@@ -128,6 +129,13 @@ class TPUConfig(
       time, but as a consequence TPUEstimator may non-deterministically
       distribute batches to different cores, rather than guaranteeing round
       robin behavior.
+    experimental_customized_tpu_infeed_outfeed_session_hook_class: This is a class
```

Comment:
API owners notes: consider renaming this `experimental_feed_hook` (and `feed_hook` later).

Reply:
> For @tensorflow/api-owners: we can merge this (with the name change), after LGTM from @bfontain.
Great, thank you François!

Reply:
Renaming is done.